### PR TITLE
Drupal 8 Webform 

### PR DESCRIPTION
**Pull request by KarinG**
and link to its new location

### CODE REVIEW

**Comment by MikeyMJCO** [comment]
An entire page to link externally seems a waste. We can just have a menu item link to the webform d8 book.

File: mkdocs.yml [hunk 1]
```diff
@@ -99,6 +99,7 @@ nav:
   - Drupal integration:
     - Drupal Users, Roles and Groups integration: integration/drupal/index.md
     - Drupal views: integration/drupal/views.md
+    - Webform (Drupal 8): integration/drupal/webform8.md
```
Comment (MikeyMJCO) [inline mkdocs.yml:102]:
```suggestion
    - Webform (Drupal 8): https://docs.civicrm.org/webform-civicrm/en/latest/
```

**Comment by KarinG** [comment]
Sure that works!

**Comment by MikeyMJCO** [comment]
If you can remove the markdown file and just leave the mkdocs.yml change!

**Pull request merged**